src/main/java/edu/ie3/datamodel/io/factory/input/SwitchInputFactory.java

```diff
@@ -34,6 +34,19 @@ protected SwitchInput buildModel(
       OperatorInput operator,
       OperationTime operationTime) {
     final boolean closed = data.getBoolean(CLOSED);
+
+    try {
+      final int parallelDevices = data.getInt(PARALLEL_DEVICES);
+      log.warn(
+          "Found a `SwitchInput` with the id `{}` that specifies `parallelDevices` with a value of `{}`. Because switches cannot be parallel, the given value is ignored!",
```

Comment:
I think "switches cannot be parallel" is not true, it's more that we do not specify the amount of parallel lines for switches, as they are supposed to have zero impedance. Maybe you can find a way to improve the warning message a bit.